src/main/java/seedu/nova/model/event/Lesson.java

```diff
@@ -11,8 +11,9 @@
 public class Lesson extends Event {
     private DayOfWeek day;
 
-    public Lesson(String desc, String venue, LocalTime startTime, LocalTime endTime, DayOfWeek day) {
-        super(desc, venue, startTime, endTime);
+    public Lesson(String description, String venue, LocalTime startTime, LocalTime endTime,
+                  DayOfWeek day) {
+        super(description, venue, startTime, endTime, day);
         this.day = day;
```

Comment:
can probably remove "this.day = day" since day is already in the super()? no need to make a new commit though